### ISSUE TITLE
Footer dark mode accessibility issue

### ISSUE THREAD

**Issue by RhysMAdey** [DiamondLightSource/sci-react-ui#108]
When in dark mode the footer fails accessibility on colour contrast

https://coolors.co/contrast-checker/ffffff-2641a2

**Comment by RhysMAdey** [comment]
Using the Diamond Them 'dark' #435184 passes accessibility
